pandas/tests/extension/test_common.py

```diff
@@ -6,14 +6,13 @@
 
 import pandas as pd
 import pandas._testing as tm
-from pandas.core.arrays import ExtensionArray
+from pandas.tests.test_common import (
+    DummyArray as _DummyArray,
```

Comment:
Kinda getting lost in the renaming, where are `DummyArray` and `DummyDtype` defined here?

Reply:
Oh nevermind, should have kept scrolling

Reply:
I think these are fine where they are. It would help if it was possible when subclassing just to give them a small naming difference like `ThisTestDummyArray(DummyArray)` instead of just `as _DummyArray`

Reply:
This will hopefully be simpler when ExtensionArray/Dtype are not abstract anymore.

Reply:
Almost no tests changed after not making ExtensionArray/Dtype abstract.